question_2.py

```diff
@@ -0,0 +1,23 @@
+class Person:
```

Comment:
Please add the exercise question at top of the script.

_"""
Add exercise question in multiline comments
"""_

Reply:
done